com.unity.render-pipelines.universal/ShaderLibrary/Shadows.hlsl

```diff
@@ -269,6 +267,26 @@ float3 ApplyShadowBias(float3 positionWS, float3 normalWS, float3 lightDirection
     return positionWS;
 }
 
+float ApplyShadowFade(float shadowAttenuation, float3 positionWS)
+{
+#if FADE_SHADOWS
+    float3 camToPixel = positionWS - _WorldSpaceCameraPos;
+    float distanceCamToPixel2 = dot(camToPixel, camToPixel);
+#ifdef _MAIN_LIGHT_SHADOWS_CASCADE
+    float shadowDist = _MainLightShadowParams.z*0.8;
```

Comment:
this can be done in C# when we compute this parameter instead of in shader. Also, please add comment explain why distance is different when using cascades.